.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node_version }}
+          node-version: ${{ matrix.node }}
```

Comment:
Fixed the CI tests to make sure that each environment uses the correct Node version. They were all using the same Node 18 because of the typo.